sandbox-backend/src/main/java/com/exadel/sandbox/team5/entity/Location.java

```diff
@@ -0,0 +1,30 @@
+package com.exadel.sandbox.team5.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Data
```

Comment:
Please use the following:
```
@Setter
@Getter
```
instead of 
`@Data`
because of BaseEntity already has @Data. It means that `hashCode` and `equals` methods were overridden in that class. And we should avoid using that annotation in child classes to avoid using all field in `hashCode` and `equals` methods.